src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/ChainPal.Apple.cs

```diff
@@ -118,7 +118,7 @@ public void Dispose()
 
         private SafeCreateHandle PreparePoliciesArray(bool checkRevocation)
         {
-            IntPtr[] policies = new IntPtr[checkRevocation ? 2 : 1];
+            Span<IntPtr> policies = checkRevocation ? stackalloc IntPtr[2] : stackalloc IntPtr[1];
```

Comment:
It's okay to put stack data in to `CFArrayCreate`. It creates a CoreFoundation array from the contents of the stack, it doesn't try to take ownership of the stack data.

We're also doing this in `SafeDeleteSslContext` as well.

https://github.com/dotnet/runtime/blob/cde8b0b6a0aa81f0af75b9adf55ebea4591ddee4/src/libraries/System.Net.Security/src/System/Net/Security/Pal.OSX/SafeDeleteSslContext.cs#L120-L122